pkg/client/restclient/OWNERS

```diff
@@ -0,0 +1,24 @@
+reviewers:
```

Comment:
@deads2k @caesarxuchao sorry I had to make the PR editable manually. I've updated the gazillion pull-request to make them editable. You should be good to go now.